### PR TITLE
Limit the length of app password names

### PR DESCRIPTION
**Pull request by nickvergessen**
Signed-off-by: Joas Schilling <coding@schilljs.com>

### CODE REVIEW

File: apps/settings/lib/Controller/AuthSettingsController.php [hunk 1]
```diff
@@ -145,6 +145,10 @@ public function create($name) {
 			return $this->getServiceNotAvailableResponse();
 		}
 
+		if (mb_strlen($name) > 128) {
+			$name = mb_substr($name, 0, 120) . '…';
```
[the remark this answers is not in the document]
Comment (nickvergessen) [reply apps/settings/lib/Controller/AuthSettingsController.php:149]: Sanity and also to not only cut of 2 characters in awkward edge cases

**Review by icewind1991 — APPROVED** [review]

**Review by miaulalala — APPROVED** [review]

**Comment by nickvergessen** [comment]
adjusted the tests

**Pull request merged**